jokes.md

```diff
@@ -0,0 +1,12 @@
+`1.(This was ranked #1): A woman gets on a bus with her baby. The bus driver says: “Ugh, that’s the ugliest baby I’ve ever seen!” The woman walks to the rear of the bus and sits down, fuming. She says to a man next to her: “The driver just insulted me!” The man says: “You go up there and tell him off. Go on, I’ll hold your monkey for you.  
+
+
+I said to the Gym instructor “Can you teach me to do the splits?” He said, “How flexible are you?” I said, “I can’t make Tuesdays.”
```

Comment:
The punctuation in this file needs some work. For instance, this line should be adjust like so:
```suggestion
I said to the gym instructor “Can you teach me to do the splits?” He said, “How flexible are you?” I said, “I can’t make Tuesdays.” haha
```